homepage/migrations/0006_review_test_data.py

```diff
@@ -15,30 +15,39 @@ def generate_data(apps, schema_editor):
         from homepage.models import Course
         from homepage.models import Review
         from homepage.models import Professor
-        from django.utils import timezone
         from django.core.files.uploadedfile import SimpleUploadedFile
+        from os import path
         from django.conf import settings
+        from datetime import datetime
+        import pytz
 
         users = AppUser.objects.all()
         courses = Course.objects.all()
         professors = Professor.objects.all()
+        image = ''
 
-        image_path = settings.MEDIA_ROOT + '/images/test_image.jpg'
-        image = SimpleUploadedFile(
-            name='new_test_image.jpg',
-            content=open(image_path, 'rb').read(),
-            content_type='image/jpeg')
+        # prevent creating a new image upon every pytest run
+        if not path.exists(settings.MEDIA_ROOT + '/images/new_test_image.jpg'):
```

Comment:
please use path.join it can prevent all the mess of concatenate the path 
```suggestion
        if not path.exists(path.join(settings.MEDIA_ROOT, '/images/new_test_image.jpg')):
```